ParabolInc/parabol
# chore(poker): use generic components for GitHub and GitLab

Fixes #6435

How to test:

- Create poker meeting
- On the scope phase play with all integrations controls to make sure nothing is broken
